src/lnp/application/payment/types.rs

```diff
@@ -392,3 +424,26 @@ impl LightningDecode for ShortChannelId {
         })
     }
 }
+
+impl Strategy for ShortChannelId {
```

Comment:
I prefer to have here `lnp::presentation::encode::Strategy` to understand which strategy we are talking about here

Reply:
That might look ugly because we need `crate` in front of it too. I will use `LnpEncodingStrategy` as a rename. 
Also, there is only one strategy used in this module. And if we keep the encoding pattern the same as now (Strict first, then LN) that's all we will need.